src/android/com/hutchind/cordova/plugins/streamingmedia/StreamingMedia.java

```diff
@@ -1,108 +1,108 @@
-package com.hutchind.cordova.plugins.streamingmedia;

```

Comment:
Please fix this file so it isn't completely recommitted

Reply:
When I did the git-clone on the main branch, these files were changed. I did not change anything. Could you help me fixing this files in the main branch?

Reply:
I'm not sure how that's possible @cesarak the master branch is the latest code. You can fix it by committing additional commits to your branch

Reply:
I'll do it all over again.

Reply:
I cloned again your main branch, and when I did the `git status`...

`cordova-plugin-streaming-media on  master [!] is 📦 v2.2.0 via ⬢ v13.12.0 $ git status`
`On branch master`
`Your branch is up to date with 'origin/master'.`

`Changes not staged for commit:`
`  (use "git add <file>..." to update what will be committed)`
`  (use "git restore <file>..." to discard changes in working directory)`
`	modified: src/android/com/hutchind/cordova/plugins/streamingmedia/StreamingMedia.java`	`modified:   www/StreamingMedia.js`

`no changes added to commit (use "git add" and/or "git commit -a")`

Reply:
That means you must have a global git configuration on your machine which is altering every file that you pull. Please remove that config it will make it impossible to contribute to open source.

Reply:
I am going to push a gitattributes file which should resolve this issue. Thanks for your patience.

Reply:
Ok all set. If you merge master back into your branch the diff should be fixed so I can see it now

Reply:
I think now is ok. If you find any error, please let me know.

Reply:
Thanks for contributing!